discovery-provider/src/queries/queries.py

```diff
@@ -729,6 +758,19 @@ def get_repost_feed_for_user(user_id):
         feed_results = sorted(
             unsorted_feed, key=lambda entry: entry[response_name_constants.activity_timestamp], reverse=True)
 
+        if "with_users" in request.args and request.args.get("with_users") != 'false':
+            user_id_list = list(set(map(lambda result: int(result["playlist_owner_id"]) if 'playlist_id' in result else int(result["owner_id"]), feed_results)))
+            users = get_users_by_id(session, user_id_list)
+            for result in feed_results:
+                if 'playlist_owner_id' in result:
+                    user = users[result['playlist_owner_id']]
```

Comment:
yeah similar comment - can't this be added in during feed construction with the conditional 'with_users'?